applications/smart-distancing/smart_distancing/__main__.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python3
+import os
+import logging
+
+import smart_distancing as sd
+
+from typing import (
+    Union,
+)
+
+logger = logging.getLogger(__name__)
+
+__all__ = ['main']
+
+def load_config(config: str) -> sd.core.ConfigEngine:
+    """
+    :return: a ConfigEngine for an absolute or relativefilename.\
+        
+    Searches config dirs in order specified by sd.DEFAULT_TO_USER_PATHS
+    """
+    logger.debug(f"load_config:config selected: {config}")
+    # if it exists as a filename, load it
+    if os.path.isfile(config):
+        return sd.core.ConfigEngine(config)
+    
+    # if it exists in the config paths, try loading it
+    if config in sd.utils.file.config_files():
+        try:
+            logger.debug(f"trying {sd.CONFIG_DIR}")
+            return sd.core.ConfigEngine(
+                os.path.join(sd.CONFIG_DIR, config))
+        except FileNotFoundError:
```

Comment:
Aha. That is the fallback logic I mentioned earlier. The primary path is searched and then the secondary, allowing the user to override the module level config by placing a .ini in `~/.smart_distancing/configs/`

I am not overly attached to it, however. Like I said, It's totally fine with me if it's removed. Less code is generally better. The only reason I added it because of the module path issue.

Reply:
Gotcha! Let's decide this later then.